tests/unit/data/aquacomms/aquacomms1.eml

```diff
@@ -0,0 +1,91 @@
+Date: Thu, 26 Aug 2021 18:22:09 +0100
+Subject: Aqua Comms Planned Outage Work ISSUE=11111 PROJ=999
+Content-Type: multipart/alternative; boundary="000000000000015b7605ca799cf2"
+
+--000000000000015b7605ca799cf2
+Content-Type: text/html; charset="UTF-8"
+Content-Transfer-Encoding: quoted-printable
+
+  <font face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2">
+    When replying, type your text above this line.
+    <hr>
+    <br>
+    <table border=3D"0" cellpadding=3D"0" cellspacing=3D"0">
+      <tbody><tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2">Dear Network to Code,<br><br></font></td>
+      </tr>
+      <tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2">This notice is being sent to notify you of t=
+he following Maintenance Event.<br><br></font></td>
+      </tr>
+
+    </tbody></table>
+    <table border=3D"0" cellpadding=3D"0" cellspacing=3D"0">
+      <tbody><tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2"><b> Ticket Number: </b></font></td>
+        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
+ face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2"> 11111</font></t=
+d>
+      </tr>
+      <tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2"><b> Scheduled Start Date &amp; Time: </b></f=
+ont></td>
+        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
+ face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2"> 22:00 12/10/202=
+0 GMT</font></td>
+      </tr>
+      <tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2"><b> Scheduled End Date &amp; Time: </b></fon=
+t></td>
+        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
+ face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2"> 08:00 13/10/202=
+0 GMT</font></td>
+      </tr>
+     =20
+      <tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2"><b> Service ID: </b></font></td>
+        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
+ face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2">111-AAA-11-11BB-=
+33333</font></td>
+      </tr>
+     =20
+    </tbody></table>
+    <br>
+    <table border=3D"0" cellpadding=3D"0" cellspacing=3D"0">
+      <tbody><tr>
+        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
+vetica, sans-serif" size=3D"2">Please note that all time stamps below will =
+appear in US Central time.<br><br></font></td>
+      </tr>
+     =20
+       =20
+        <tr>
+          <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, H=
+elvetica, sans-serif" size=3D"2">- Update - There are no new updates.<br></=
+font></td>
+        </tr>

```

Comment:
Should this go into the maintenance `summary` field instead of leaving it blank?